config/environments/local/log4j.properties

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger=DEBUG, A1
+log4j.appender.A1=org.apache.log4j.ConsoleAppender
```

Comment:
If the only difference between these files is the log level, might be cleaner to have one log4j.properties file and specify log level in settings/common.yaml.

You can do a string replace before copying the properties file into the binary, for an example we do this a few places in config/gcp/setup_gke_environment.sh with sed

Reply:
For now the only diff is the level, but over time I think they will differ more so I think we should just leave them separate.

Reply:
SGTM